src/ontology/mondo-edit.obo

```diff
@@ -167882,41 +167882,20 @@ property_value: seeAlso https://rarediseases.info.nih.gov/diseases/331/yunis-var
 subset: ordo_disease {source="Orphanet:1454"}
-synonym: "cerebellar vermis hypo/aplasia, oligophrenia, ataxia congenital, coloboma, and hepatic fibrosis" RELATED [GARD:0001410]
-synonym: "cerebellar vermis hypo/aplasia, oligophrenia, congenital ataxia, ocular coloboma, and hepatic fibrosis" EXACT [OMIM:216360]
-synonym: "cerebellar vermis hypoplasia-oligophrenia-congenital ataxia-coloboma-hepatic fibrosis" EXACT [Orphanet:1454]
-synonym: "COACH syndrome" BROAD [OMIM:216360, Orphanet:1454]
-synonym: "gentile syndrome" EXACT [Orphanet:1454]
-synonym: "Joubert syndrome with congenital hepatic fibrosis" EXACT [OMIM:216360, Orphanet:1454]
-synonym: "Joubert syndrome with hepatic defect" EXACT [Orphanet:1454]
-synonym: "JS-H" EXACT [Orphanet:1454]
-xref: DOID:0111589 {source="MONDO:equivalentTo"}
-xref: GARD:0001410 {source="MONDO:equivalentTo"}
 xref: ICD10CM:Q04.3 {source="Orphanet:1454/attributed", source="Orphanet:1454/ntbt", source="Orphanet:1454"}
-xref: MESH:C536430 {source="MONDO:equivalentTo"}
-xref: OMIM:216360 {source="Orphanet:1454", source="MONDO:equivalentTo", source="Orphanet:1454/e"}
-xref: Orphanet:1454 {source="MONDO:equivalentTo", source="OMIM:216360"}
-xref: SCTID:721847002
-xref: UMLS:C1857662 {source="Orphanet:1454", source="MONDO:equivalentTo", source="MONDO:ncbi_mim2gene_medline", source="Orphanet:1454/e", source="OMIM:216360"}
-is_a: MONDO:0015369 {source="Orphanet:1454"} ! Joubert syndrome and related disorders
-is_a: MONDO:0015508 {source="MONDO:Redundant", source="Orphanet:1454"} ! genetic parenchymatous liver disease
-is_a: MONDO:0021147 {source="MONDO:Entailed", source="Orphanet:1454", source="Orphanet:1454/inferred"} ! disorder of development or morphogenesis
-is_a: MONDO:0100349 {source="https://orcid.org/0000-0001-5208-3432"} ! COACH syndrome
-intersection_of: MONDO:0100349 ! COACH syndrome
-intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/28396 ! TMEM67
 relationship: excluded_subClassOf MONDO:0000001 {source="Orphanet:1454"} ! disease or disorder
 relationship: excluded_subClassOf MONDO:0015160 {source="Orphanet:1454"} ! multiple congenital anomalies/dysmorphic syndrome-variable intellectual disability syndrome
 relationship: excluded_subClassOf MONDO:0017118 {source="Orphanet:1454"} ! obsolete syndrome with a cerebellar malformation as major feature
 relationship: excluded_subClassOf MONDO:0043007 {source="Orphanet:1454"} ! obsolete genetic multiple congenital anomalies/dysmorphic syndrome-variable intellectual disability syndrome
-relationship: has_characteristic MONDO:0021136 {source="MONDO:0015114", source="MONDO:0015960"} ! rare
 property_value: confidence "3.7222222222222223" xsd:double
 property_value: IAO:0000233 https://github.com/monarch-initiative/mondo/issues/4986 xsd:anyURI
-property_value: IAO:0006012 "2022-10-01" xsd:string
+is_obsolete: true
+consider: MONDO:0100349
```

Comment:
I removed this consider

Reply:
how should we record that the terms were obsoleted and replaced by 2 terms? (only 1 'replaced by; is allowed -I think)

Reply:
added 2 "replaced by" since this term is a split